Makefile

```diff
@@ -3,8 +3,11 @@ unit:
 	behave --tags=$(UNITS) test -f progress2
 
 INTEGRATIONS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @dryrun or @dryrun.testing or @indexer or @indexer.231 or @indexer.applications or @kmd or @rekey or @send.keyregtxn or @send"
-integration:
+integration: blackbox
 	behave --tags=$(INTEGRATIONS) test -f progress2
 
+blackbox:
+	pytest x/blackbox
```

Comment:
Instead of just running blackbox tests, it would be more useful to make an entry to run all local Python tests, which include the blackbox tests.

Reply:
I'm not sure what you mean here. This is not only a local test, but is a new integration test. Do you mean that it should be renamed, or that I should also add `test_unit.py` and `test_integration.py` to a section that looks something like:

```Makefile
python-specific:
    pytest x/blackbox unit_test.py integration_test.py
```
?

Reply:
Ok I stewed a bit on this, I have a proposal. Restructure the test files as follows into new directories:
* `test/local`
  * `test/local/unit` 
    * `test_unit.py` (rename to `unit_test.py` ?)
  * `test/local/integration`
    * `test/local/testnet.py` (rename to `clients.py` ?)
    * `test_integration.py` (rename to `integration_test.py` ?)
      * NOTE this test is getting its client info from `examples/tokens`. Perhaps it should get it instead from `testnet.py`
    *  `blackbox_test.py`
    *  `dryrun_mixin_docs_test.py` (if we end up keeping it)
The now orphaned `README.md` could move to `algoskd/testing/README.md`

Reply:
@tzaffi I see you decided against making the `test/cucumber` folder, why was that? I still recommend it for clarity because a lot of the cucumber setup is strange and it's best not to mix that with "normal" tests